skeleton/preload.js

```diff
@@ -290,20 +290,37 @@ const Desktop = new (class {
     getResponseEventName(module, event) {
         return `${this.getEventName(module, event)}___response`;
     }
-})();
 
+    asJSON() {
+        return {
+            electron: { webFrame: { getZoomFactor: () => 1 } },
+            fetch: (...args) => this.fetch(...args),
+            fetchAsset: (...args) => this.fetchAsset(...args),
+            fetchFile: (...args) => this.fetchFile(...args),
+            getAssetUrl: (...args) => this.getAssetUrl(...args),
+            getEventName: (...args) => this.getEventName(...args),
+            getFileUrl: (...args) => this.getFileUrl(...args),
+            on: (...args) => this.on(...args),
+            once: (...args) => this.once(...args),
+            removeAllListeners: (...args) => this.removeAllListeners(...args),
+            removeListener: (...args) => this.removeListener(...args),
+            respond: (...args) => this.respond(...args),
+            send: (...args) => this.send(...args),
+            sendGlobal: (...args) => this.sendGlobal(...args),
+            setDefaultFetchTimeout: (...args) => this.setDefaultFetchTimeout(...args)
+        };
+    }
+})();
 
 process.once('loaded', () => {
     if (process.env.NODE_ENV === 'test') {
         global.electronRequire = require;
         global.process = process;
     }
 
-    Desktop.electron = [];
-
     exposedModules.forEach((module) => {
         Desktop.electron[module] = require('electron')[module];
     });
-
-    global.Desktop = Desktop;
+    // exposeInMainWorld support only plain object with methods declared on first level
+    contextBridge.exposeInMainWorld('Desktop', Desktop.asJSON());
```

Comment:
exposing electron module `webFrame` cause `Electron contextBridge recursion depth exceeded.  Nested objects deeper than 1000 are not supported` error. Now integration test pass because I fake it in line 296 `electron: { webFrame: { getZoomFactor: () => 1 } },`

Reply:
> Well, the tests are green, so that is a good first step.

Jan (@StorytellerCZ ) which other steps are needed to commit this to a RC branch so we can test it out?

Reply:
I will do an rc release for 3.2, if I get some time in the next few days.

If not, then ping me on Monday here:
https://www.youtube.com/watch?v=8s65DpKSVeQ

Reply:
@StorytellerCZ you are going to solve issue with exposing electron modules, right? Otherwise I propose to remove this functionality from meteor-desktop, make breaking change and put in release notes information about it.

Reply:
> I will do an rc release for 3.2, if I get some time in the next few days.
> 
> If not, then ping me on Monday here: https://www.youtube.com/watch?v=8s65DpKSVeQ

Hi Jan,

can you please do a RC release?

Just got a notification from Snyk that there’s a vulnerability for Electron 11.5.0: https://security.snyk.io/vuln/SNYK-JS-ELECTRON-6097142


Thank you.

Reply:
Published communitypackages:meteor-desktop-bundler@3.2.0-rc.0.
Published communitypackages:meteor-desktop-watcher@3.2.0-rc.0.
@meteor-community/meteor-desktop@3.2.0-rc.0 under the `preview` tag.